pkg/coscheduling/coscheduling.go

```diff
@@ -203,27 +254,31 @@ func (cs *Coscheduling) Unreserve(ctx context.Context, state *framework.CycleSta
 	})
 }
 
-// GetPodGroupLabels will check the pod if belongs to some podGroup. If so, it will return the
-// podGroupName、minAvailable of podGroup. If not, it will return "" as podGroupName.
+// GetPodGroupLabels will check the pod if belongs to a  PodGroup. If so, it will return the
+// podGroupName、minAvailable of the PodGroup. If not, it will return the pod name and 0.
```

Comment:
`、` is not an ASCII character

```suggestion
// podGroupName and minAvailable of the PodGroup. If not, it will return the pod name and 0.
```